extension/sidebar-pane.js

```diff
@@ -38,7 +38,8 @@ async function _updateSelectedNode(selectedNode) {
 }
 
 async function _updateSubtree(selectedNode) {
-  const ulEl = document.querySelector("#subtree ul");
+  const subtreeEl = document.getElementById("subtree");
+  const ulEl = subtreeEl.querySelector("ul");
```

Comment:
For consistency reasons, let's call all these variables with the same pattern. So either `nameEl`, or `tagEl`.
Here you have `subtreeEl` which is a logical name followed by the El suffix. But then you have `ulEl` which is a tag name followed by the El suffix.
I advise renaming `ulEl` to `issueListEl`.